e2e/po/app.skip.ts

```diff
@@ -34,7 +34,8 @@ export class SkipLogin {
   }
 
   getAppStatus() {
-    cy.get('#app > app-root > div > app-navbar > nav > div.navbar-menu .icon').should('be.visible')
+    cy.wait(2000); // slight delay before checking app status
+    cy.get('app-navbar .navbar-menu .icon').should('be.visible')
```

Comment:
copy selector gives "#hover-link div span.icon" or just ""#hover-link div span" should be tried? or we can give a special ID to this and just get that ID.